data/brands/amenity/music_school.json

```diff
@@ -35,7 +35,28 @@
         "amenity": "music_school",
         "brand": "School of Rock",
         "brand:wikidata": "Q7756376",
-        "name": "School of Rock"
+        "brand:zh": "搖滾教室",
```

Comment:
Hello! This entry containing many locationSets is shared by the above regions. If you need one containing Chinese, you should remove Taiwan from the locationSet, create an entry containing only Taiwan, and add these three tags to the new entry.

您好！这个包含诸多locationSet的条目是被上述地区共享的，若需要一个包含中文的，您应将台湾从locationSet中移除，并创建一个仅包含台湾的条目，在新条目中添加这三个标签。

Reply:
Sorry, no, it's because the original one is closer to be shared by "global or multi-regions". Take the classic example McDonald's (at https://github.com/osmlab/name-suggestion-index/blob/main/data/brands/amenity/fast_food.json )(search Q38076) and you will find that there is a 001 version that excludes many countries, and then the excluded countries and regions will have their own versions, for example, in Ukraine, there is `name:uk=МакДональдз` in Ukrainian , but won't have the Japanese `name:ja=マクドナルド`. For those area not explicit listed, they fall back to use that just English entry.

不不不，是因为原来的更接近一个“全球或若干地区”共享的，就拿本项目较为经典的麦当劳（在 https://github.com/osmlab/name-suggestion-index/blob/main/data/brands/amenity/fast_food.json 搜索 Q38076）会发现有一个001但是exclude了很多国家地区的版本，然后被exclude的国家和地区会有自己的版本，比如在乌克兰的就带有乌语的МакДональдз，而不会带有日语的マクドナルド。如果都找不到那就回落到那个只剩英文的条目了。

---

这个地方我已经改好了